site/content/en/docs/tasks/troubleshooting/troubleshooting_provreq.md

```diff
@@ -125,25 +125,17 @@ The states transitions are as follow:
 
 If Kueue did not create a Provisioning Request for your job, try checking the following requirements:
 
-### a. Ensure the Kueue's controller manager enables the `ProvisioningACC` feature gate
+### a. Verify ProvisioningRequest support is available
 
-Run the following command to check whether your Kueue's controller manager has enabled the `ProvisioningACC` feature gate:
+Since Kueue v0.13.0, ProvisioningRequest support is Generally Available (GA) and always enabled. For older versions of Kueue (v0.7.0 to v0.12.x), the feature was enabled by default via the `ProvisioningACC` feature gate.
```

Comment:
```suggestion
Since Kueue v0.14.0, ProvisioningRequest support is Generally Available (GA) and always enabled. For older versions of Kueue (v0.7.0 to v0.13.x), the feature was enabled by default via the `ProvisioningACC` feature gate.
```

Reply:
done.